sycl/test-e2e/DeviceLib/assert-windows.cpp

```diff
@@ -1,9 +1,5 @@
 // REQUIRES: cpu,windows
 //
-// FIXME: OpenCL CPU backend compiler crashes on a call to _wassert.
-// Disable the test until the fix reaches SYCL test infrastructure.
-// XFAIL: *
```

Comment:
Just wondering - if this test passes actually, why we didn't catch the "xfailed tests passed" previously?

Reply:
Yeah, it seems I overlooked that the test is unsupported, I thouhght it had passed.
I will add a jira ticket number.

Reply:
This test is passing with the latest compiler. I will remove the XFAIL.
FYI @aelovikov-intel